examples/typescript/simple_sponsored_transaction.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Example to submit a simple sponsored transaction where Alice transfers APT coin to Bob
+ * with a sponser account to pay for the gas fee
+ */
+import { Account, Aptos, U64 } from "aptos";
+
+const ALICE_INITIAL_BALANCE = 100_000_000;
+const SPONSER_INITIAL_BALANCE = 100_000_000;
+const BOB_INITIAL_BALANCE = 0;
+const TRANSFER_AMOUNT = 10;
+
+const example = async () => {
+  console.log(
+    "This example will create three accounts (Alice, Bob and Sponser), fund Alice and Sponser, transfer between Alice and Bob with Sponser to pay the gas fee.",
+  );
+
+  // Setup the client
+  const aptos = new Aptos();
+
+  // Create three accounts
+  const alice = Account.generate();
+  const bob = Account.generate();
+  const sponser = Account.generate();
+
+  // Variables to hold Alice and Sponser accounts address
+  const aliceAddres = alice.accountAddress.toString();
+  const bobAddress = bob.accountAddress.toString();
+  const sponserAddress = sponser.accountAddress.toString();
+
+  console.log("=== Addresses ===\n");
+  console.log(`Alice's address is: ${aliceAddres}`);
+  console.log(`Bob's address is: ${bobAddress}`);
+  console.log(`Sponser's address is: ${sponserAddress}`);
+
+  // Fund Alice and Sponser accounts
+  await aptos.fundAccount({ accountAddress: alice.accountAddress.toString(), amount: ALICE_INITIAL_BALANCE });
+  await aptos.fundAccount({ accountAddress: sponser.accountAddress.toString(), amount: SPONSER_INITIAL_BALANCE });
+
+  // Show account balances
+  const aliceBalanceBefore = await aptos.getAccountCoinsData({ accountAddress: aliceAddres });
+  const sponserBalanceBefore = await aptos.getAccountCoinsData({ accountAddress: sponserAddress });
+
+  console.log("\n=== Balances ===\n");
+  console.log(`Alice's balance is: ${aliceBalanceBefore[0].amount}`);
+  console.log(`Bobs's balance is: ${BOB_INITIAL_BALANCE}`);
+  console.log(`Sponser's balance is: ${sponserBalanceBefore[0].amount}`);
+
+  if (aliceBalanceBefore[0].amount !== ALICE_INITIAL_BALANCE) throw new Error("Alice's balance is incorrect");
```

Comment:
Fyi if there's an assert library or something you think is better, you can add it.  I just wanted to keep dependencies down / keep it simple

Reply:
I actually find throwing is nicer and enough, expect/assert doesnt give nice info